realm/realm-library/src/syncIntegrationTest/java/io/realm/PermissionManagerTests.java

```diff
@@ -699,6 +699,43 @@ public void onError(ObjectServerError error) {
         pm3.acceptOffer(offerToken, callback);
     }
 
+    @Test
+    @RunTestInLooperThread
+    public void getCreatedOffers() {
+        final String offerToken = createOffer(user, "test", AccessLevel.WRITE, null);
+        PermissionManager pm = user.getPermissionManager();
+        looperThread.closeAfterTest(pm);
+
+        pm.getCreatedOffers(new PermissionManager.OffersCallback() {
+            @Override
+            public void onSuccess(RealmResults<PermissionOffer> offers) {
+                RealmResults filteredOffers = offers.where()
+                        .equalTo("token", offerToken)
+                        .findAllAsync();
```

Comment:
Because I wanted to avoid race conditions on the server as the offers need to be downloaded to the other users `__management` realm